scripts/gcp-onboarding/README.md

```diff
@@ -96,26 +92,29 @@ If you have more than 400 projects in the GCP organization then try to increase
 1. If you are onboarding a certain number of projects then create a .csv file, by running the below command on the cloud shell you can list all the projects in a .csv file.
 
 	```bash
-	gcloud alpha asset list --organization=<GCP_ORGANIZATION_ID> --content-type=resource --asset-types=cloudresourcemanager.googleapis.com/Project --format="csv(resource.data.projectId,resource.data.name)" > projlist.csv
+	gcloud alpha asset list --organization=<GCP_ORGANIZATION_ID> --content-type=resource --asset-types=cloudresourcemanager.googleapis.com/Project --filter=resource.data.lifecycleState=ACTIVE --format="csv(resource.data.projectId,resource.data.name)" > projectlist.csv
 	```
 	Updates CSV file as per requirement.
 
-2. Execute the below commands to enable service APIs for multiple GCP projects present in CSV file
+2. Execute the below commands to enable Cloud APIs for multiple GCP projects present in CSV file
 
 	```bash
 	./enable-gcp-api.sh -s <SERVICE_ACCOUNT_PROJECT_ID> -c <PROJECT_LIST.CSV>
 	```
 
+**Note:**
+Cloud APIs are also enabled on project in which Service Account is created.
```

Comment:
The above script enables the required Cloud APIs on the project in which the service account got created during step 1

Reply:
Updated the changes